docs/man/rpm-queryformat.7.scd

```diff
@@ -0,0 +1,290 @@
+RPM-QUERYFORMAT(7)
+
+# NAME
+*rpm-queryformat* - RPM query format
+
+# SYNOPSIS
+
+%[_WIDTH_]{_TAG_[:_FORMAT_]}
+
+# DESCRIPTION
+
+The *rpm* query format is a static string modeled after the standard *printf*(3)
+format. It uses *rpm* header tags enclosed in curly braces in place of
+conversion specifiers, and allows you to specify what package information should
+be printed in query operations and how it should be formatted.
+
+Each _TAG_ in the string may specify an output _FORMAT_ to further customize the
+way it is displayed. Special characters such as newlines or tabs (with the
+exception of *\\0*), as well as the *printf*(3) field _WIDTH_, are also
+supported.
+
+The string is accepted as an argument to the *--queryformat* option in *rpm*(8)
+and in runtime scriptlets using the *-q* option in *rpm-spec*(5) files.
+
+## Tags
+
+Every *rpm* package contains a set of meta-data records describing its
+properties, such as the name, version or file list. Each record is either a
+plain string or number, or an array of either type. These records are stored
+inside a package, in a section called the _header_, and are identified by unique
+names called _tags_. The individual records are also referred to as _tag data_.
+
+You can use these tags in the query format to display the individual properties
+of a package. To obtain a list of all the tags supported by the version of *rpm*
+installed on your system, run the following command:
+
+```
+rpm --querytags
+```
+
+This will print a list of tags, one per line, for example:
+
+```
+BUILDHOST
+BUILDTIME
+DESCRIPTION
+EPOCH
+INSTALLTIME
+NAME
+RELEASE
+SIZE
+SUMMARY
+VERSION
+```
+
+Each tag is case-insensitive and also has an implicit variant with the *RPMTAG_*
```

Comment:
"Each tag is case-insensitive" is a bit funny expression. Maybe just "Tags are case-insensitive"?